crates/evm/traces/src/identifier/local.rs

```diff
@@ -34,7 +34,10 @@ impl TraceIdentifier for LocalTraceIdentifier<'_> {
                     .iter()
                     .filter_map(|(id, (abi, known_code))| {
                         let score = diff_score(known_code, code);
-                        if score < 0.1 {
+                        // Note: the diff score can be inaccurate for small contracts so we're using
+                        // a relatively high threshold here to avoid filtering out too many
+                        // contracts.
+                        if score < 0.85 {
```

Comment:
super small nit, but maybe we can lift this to a constant and document it?

Reply:
I think that's fine here, I also think this entire logic deserves another look because we only do this to filter out candidates for decoding matches, the better approach would be to just get __all__ function selectors